src/translations/sqlb_zh.ts

```diff
@@ -6639,12 +6639,12 @@ Are you sure you want to proceed?</source>
     <message>
         <location filename="../SqlExecutionArea.cpp" line="37"/>
         <source>Ctrl+PgUp</source>
-        <translation type="unfinished">Ctrl+PgUp</translation>
+        <translation>Ctrl+PgUp</translation>
```

Comment:
```suggestion
        <translation></translation>
```
You don't need to translate keystrokes. Reference: https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translations#dont-translate-key-names-in-shortcuts